app/src/org/commcare/views/widgets/GeoPointWidget.java

```diff
@@ -213,9 +196,9 @@ public void setBinaryData(Object answer) {
         String[] sa = s.split(" ");
         mAnswerDisplay.setText(
                 StringUtils.getStringSpannableRobust(getContext(), R.string.latitude) +
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/82427d28695ad1888b353511341c08d9209c1a85/app/src/org/commcare/views/widgets/GeoPointWidget.java#L198"></span>Do not concatenate text displayed with <code>setText</code>. Use resource string with placeholders.</td>
    </tr>
  </tbody>
</table>